source/Effect.h

```diff
@@ -71,6 +73,9 @@ class Effect : public Body {
 	int lifetime = 0;
 	int randomLifetime = 0;
 
+	// If set, this effect scales with the zoom of the ship engines bay that placed it.
```

Comment:
I don't think we call them "bays". Maybe "engine point"?
```suggestion
	// If set, this effect scales with the zoom of the ship engines point that placed it.
```
Although, to better match my other comment:
```suggestion
	// If set, this effect scaled can be scaled based on the "zoom" of the object it is used on.
	// For example, engine points for afterburner effects.
```